### ISSUE TITLE
Footnote highlighting

### ISSUE THREAD

**Issue by not-AMM** [mb21/panwriter#101]
Are the footnotes purposedly highilighted like this: 

![imagen](https://user-images.githubusercontent.com/43765089/155261921-47ac3778-3ae8-44eb-993b-1345cee1d2bc.png)

They work in preview, the editor works great! The only problem I've found is footnotes looking like that. This is not urgent, just to let you know. 

P.S.: just a suggestion, after opening the metadata menu, it would be better to have a close button in the upper right corner. Btw, thanks for all your hard work, I'm enjoying Panwriter a lot!

**Comment by mb21** [comment]
> Are the footnotes purposedly highilighted like this?

No, it's a artefact of how CodeMirror parses markdown... if you can find a way to change this in [Editor.css](https://github.com/mb21/panwriter/blob/master/src/components/Editor/Editor.css) without breaking anything else that would be great, but not sure it's possible.

> after opening the metadata menu, it would be better to have a close button in the upper right corner

to replace the back button in the upper left?
